features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -881,8 +881,8 @@
   "myaccount.ui.app_copyright": "WSO2 Identity Server",
   "myaccount.ui.app_title": "My Account | WSO2 Identity Server",
   "myaccount.ui.app_name": "My Account",
-  "myaccount.ui.app_logo_path": "/assets/images/logo.svg",
-  "myaccount.ui.product_name": "Identity Server",
+  "myaccount.ui.app_logo_path": "/assets/images/logo-full.svg",
```

Let's use `logo.svg` here and refactor the login portals to stop using text to resolve product name. 
```suggestion
  "myaccount.ui.app_logo_path": "/assets/images/logo.svg",
```

---

+1 resolved in commit bcc0138